qutebrowser/config/configdata.yml

```diff
@@ -1994,6 +1994,16 @@ input.mode_override:
   supports_pattern: true
   desc: Mode to change to when focusing on a tab/URL changes.
 
+input.better_input_detection:
```

Comment:
I'd prefer a more descriptive name like `use_ime` or somesuch. If we change things in the future (and perhaps come up with more refinements or an other method entirely, "better" can be a confusing term - and given it being experimental and having its issues as well apparently, a more neutral term seems like a better fit too.